infrastructure/pre_req.sh

```diff
@@ -77,7 +77,12 @@ echo "Namespace: $namespace"
 if aws s3api head-bucket --bucket "${bucket_name}" --region "${aws_region}" >/dev/null 2>&1; then
     echo "S3 bucket ${bucket_name} already exists"
 else
-    if aws s3api create-bucket --bucket "${bucket_name}" --region "${aws_region}" --create-bucket-configuration LocationConstraint="${aws_region}" && \
+    # A location constraint for "us-east-1" returns an error. For "us-east-1", we omit --create-bucket-configuration
+    if [[ -n "$aws_region" ]] && [[ "$aws_region" != "us-east-1" ]]; then
```

Comment:
What are we doing here and why do we need it here? @sdebruijn

Reply:
We leave out the LocationConstraint if the region is set to us-east-1. The LocationConstraint cannot be set with us-east-1 as parameter. 
I've updated the pull request description with a link to the AWS documentatation for create-bucket.

Reply:
okay